src/views/service/BotServiceStatus.tsx

```diff
@@ -16,11 +16,22 @@ const ServicePage: React.FC = () => {
   const isRequesting: boolean = useSelector((state: IAppState) =>
     selectRequesting(state, [BotServiceActions.REQUEST_BOT_SERVICE])
   );
+  const isStarting: boolean = useSelector((state: IAppState) =>
+    selectRequesting(state, [BotServiceActions.REQUEST_START_SERVICE])
+  );
+  const isStoping: boolean = useSelector((state: IAppState) =>
+    selectRequesting(state, [BotServiceActions.REQUEST_STOP_SERVICE])
+  );
 
   React.useEffect(() => {
     // Array of one virtual machine will be fetched
-    dispatch(getBotServiceAsync());
-  }, []);
+    if (!isStarting && !isStoping) {
```

Comment:
**nitpick(blocking)**: 
- We are enabling the polling only if the service is not starting or stopping, but what would happen if we have multiple services?
- We don't use setTimeOut in components. We have to use the useInterval hook.